SparkPost/2web2ui
# FE-1076 - Address intermittently crashing Cypress runner in CircleCI

[FE-1076](https://jira.int.messagesystems.com/browse/FE-1076)

### What Changed
- I _tried_ using the Cypress Docker image to address the browser crashing issue, but no luck - not to mention they don't support all Node versions (in particular, the version our application is pegged to)
- I _tried_ using the Cypress Circle CI orb - this defaulted to the Cypress Docker image that caused the same Node version incompatibility issue
- Instead, I updated a few tests by removing unnecessary network requests
- Updated a flaky templates test to use a different means of checking local storage that *should* be more flake-proof
- Removed extraneous request stubbing in `changeBillingPlanPage.spec.js`
- Removed test for required field requiring validation on each form field
- Updated health score dashboard tests to assert more frequently, reducing test flakiness. See: https://docs.cypress.io/guides/core-concepts/retry-ability.html#Alternate-commands-and-assertions
- Updated summary reports tests to use new `findByDataId` method. Leveraged `DEFAULT_TO_HIBANA` env variable to check for different data ID due to changing location of elements

### How To Test
- Run this branch repeatedly in CircleCI
- Verify that the browser doesn't crash in CircleCI

### To Do
- [X] Run the pipeline with all steps enabled
